pkg/deploy/deployment_keycloak.go

```diff
@@ -205,7 +214,7 @@ func NewKeycloakDeployment(cr *orgv1.CheCluster, keycloakPostgresPassword string
 			},
 		}
 	}
-	command := addCertToTrustStoreCommand + " && " + changeConfigCommand + " && /opt/jboss/docker-entrypoint.sh -b 0.0.0.0"
+	command := addCertToTrustStoreCommand + " && " + changeConfigCommand + " && /opt/jboss/docker-entrypoint.sh -b 0.0.0.0 -c standalone.xml"
```

Comment:
why new standalone.xml?

Reply:
Because in the last `jboss/keycloak` docker image we upgraded to, the default behavior is now to run Keycloak with the `standalone-ha` profile. This brought certificate problems since the Che operator still imports the required certificates into the `standalone` profile. Additionally, the *ha* profile is expected to consume more resources, since it is configured to setup a cluster of Keycloak servers, which is not needed nor supported in our case (at least now).